packages/sql-mssql/package.json

```diff
@@ -40,11 +40,13 @@
   "devDependencies": {
     "@effect/platform": "workspace:^",
     "@effect/sql": "workspace:^",
+    "@opentelemetry/semantic-conventions": "^1.24.1",
     "effect": "workspace:^"
   },
   "peerDependencies": {
     "@effect/platform": "workspace:^",
     "@effect/sql": "workspace:^",
+    "@opentelemetry/semantic-conventions": "^1.22",
```

Comment:
I think we can make it a normal dependency, as all the usage is internal.

Reply:
Thanks! Addressed in https://github.com/Effect-TS/effect/pull/2712/commits/170080dbaa4aebfef4aad3b934213f48edd95085